prime-router/metadata/schemas/direct/imagemover-covid-19.schema

```diff
@@ -19,3 +19,13 @@ elements:
   - name: placer_order_id
     csvFields: [ { name: specimenId } ]
 
+  # Force remove the value of the ordering provider name
+  - name: ordering_provider_first_name
+    default:
+    defaultOverridesValue: true
+
+  # Force remove the value of the ordering provider name
+  - name: ordering_provider_last_name
+    default:
+    defaultOverridesValue: true
```

Comment:
Is there a way to separate this behavior from the `default` behavior? So maybe using a key like `value`, `constant`, or `override`?

And then the code below becomes something separate like:
```
if (override != null) {
  retVal = override
}
```

Reply:
Well, I started out by attempting to create a "blank the darn thing out()" mapper, but failed, because getting a mapper to return a null or empty string, and have it actually take effect, would have required a giant risky change I was not willing to make.

I still think the mapper is the best approach (in fact, I would implement defaults as a default() mapper.  But it would require a lot of testing.

Reply:
makes sense, the full mapper move is huge. I find the `defaultOverridesValue` thing paired with `default` to be confusing, it's overloading the purpose of a `default`. Reading through the comments though I think I get it, this feature also allows the `defaultOverrides` to to override a value now. If that's part of this then sounds good. If it's not then I think a separate behavior of `override` or `constant` or something is clearer.